### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by asvetlov**
Minor code and documentation cleanups

### CODE REVIEW

File: aiohttp/helpers.py [hunk 1]
```diff
@@ -514,11 +514,11 @@ def _format_line(self, request, response, time):
     def log(self, request, response, time):
         """Log access.
 
-        :param message: Request object. May be None.
-        :param environ: Environment dict. May be None.
-        :param response: Response object.
-        :param transport: Tansport object. May be None
-        :param float time: Time taken to serve the request.
+        message: Request object. May be None.
```
Comment (hellysmile) [inline aiohttp/helpers.py]: these parameters is not relevant actually

**Comment by kxepal** [comment]
I'm not sure about global sphinx markup rid. Even if not for autodocs, it's very useful for one more thing: type hints. PyCharm, for instance, can use that information if no annotations provided. And when you code, it's easy to note what kind of objects are required.

I would recommend postpone that change and made it with migration to type annotations to not loose anything.

**Comment by asvetlov** [comment]
@kxepal markup is not in actual state very often and present for very small subset of API.
We need review it anyway.
I don't think that removed sphinx markup makes any sense.

**Comment by kxepal** [comment]
@asvetlov ok then.

**Pull request merged**